profiles/pdf.md

```diff
@@ -0,0 +1,51 @@
+# PDF Profile
+
+**Editors:**
+
+* Michael Benowitz ([NYPL](https://www.nypl.org))
+
+**Participate:**
+
+* [GitHub readium/webpub-manifest](https://github.com/readium/webpub-manifest)
+* [File an issue](https://github.com/readium/webpub-manifest/issues)
+
+## Introduction
+
+The display of static PDF files can be governed by the standard Webpub Manifest guidelines, as the format was specifically conceived of to provide a constent, portable format. However there are many optional features, mainly pertaining to accessibility and preservation, that can be communicated to users. These features may be used in a client context or used in other ways by consuming applications.
+
+This profile is intended to ensure that full functionality of PDF resources is documented in Webpub Manifests, and conversely ensuring that PDFs with limited functionality are documented.
+
+## 1. Declaring conformance with PDF Profile
+
+In order for a Webpub Manifest to conform to this profile it <strong class="rfc">must</strong> include a `conformsTo` key in the `metadata` section, with `http://readium.org/webpub-manifest/profiles/pdf` as the value.
+
+## 2. `readingOrder` resource restrictions
+
+A conforming Webpub Manifsest <strong class="rfc">must</strong> only refererence PDF documenents with a media type of `application/pdf` in the `readingOrder` section.
+
+## 3. PDF Encryption
+
+If a Webpub Manifest contains encrypted resources, those resources <strong class="rfc">must</strong> include an [encryption object](https://github.com/readium/webpub-manifest/blob/master/modules/encryption.md#encryption-object) in the `properties` object attached to each `Link` object.
+
+
+## 4. Link Parameters
+
+A Manifest <strong class="rfc">may</strong> include the following parameters in the `href` attribute of elements in the `readingOrder` and `toc` sections.
+
+| Key   | Semantics | Type     | Values    | 
+| ----- | --------- | -------- | --------- | 
+| [start](#start) | Specifies the initial page of the PDF to display when displaying this resource  | Integer  | 1 to (page count of current resource)  | 
```

Comment:
Allowing fragments in the reading order might introduce a lot of complexities in the toolkits. We'll need to talk about this, would you be able to come to one of our weekly Zoom meetings ([next one](https://24timezones.com/fr/event?txt=Weekly%20Readium%20Meeting&st=20220713T1730&l=c1195))? 

That being said, you could use the fragment identifier `page` instead of `start`, as it is widely supported for PDF:

> The list of PDF-open parameters and the action they imply is:
> * `page=<pagenum>` Open the specified (physical) page.
>
> https://datatracker.ietf.org/doc/html/rfc3778#section-3

Reply:
Agree with @mickael-menu, we want to avoid fragments in the reading order.

This use case isn't restricted to PDF either and would be better addressed in the main spec IMO:

- if this matches a resource, specifying `start` as a `rel` in the `readingOrder` is enough
- if this matches a fragment of a resource, then a new Link Object in `links` would be the better option

Reply:
Thanks for the feedback, would be happy chat about this! I may not be able to make next weeks, but will try to attend one soon. I'm curious about the fragment discussion and what we can do to support features like this without introducing too much complexity.

@HadrienGardeur can you explain your second point regarding representing fragments as a new object in `links`? I'm not sure I follow how this addresses the functionality we're looking for.